documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,56 @@
+== External Trigger on the Global Shutter Camera
+The global shutter camera can be triggered externally by pulsing the XTR pin on the board.
```

Comment:
I'm _guessing_ that "XTR" is an abbreviation of "eXternal TRigger"? Is it worth explaining that?